godtools/GodToolsConstants.swift

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
I'm very old school haha, I set the GT on all files, but let me know if I should change it